docs/pages/versions/v33.0.0/sdk/gesture-handler.md

```diff
@@ -12,8 +12,40 @@ This API is pre-installed in [managed](../../introduction/managed-vs-bare/#manag
 
 ## API
 
+If you want to import all available gesture handlers:
```

Comment:
Could you please remove this top section and leave just the second? (We don't want to have to maintain both sections, and the second one implies the first is possible -- it's just plain JavaScript and nothing specific to Expo.)